packages/button/package.json

```diff
@@ -1,13 +1,16 @@
 {
   "name": "@constellation-components/button",
   "author": "maecapozzi <maecapozzi@gmail.com>",
-  "version": "1.0.1-alpha.5",
+  "version": "1.0.2",
```

Comment:
I accidentally published a patch instead of a beta release. 😢 Automation will hopefully help a lot with this.